XIVComboVX/Combos/WHM.cs

```diff
@@ -82,20 +83,15 @@ internal class WhiteMageStone: CustomCombo {
 		
 		if (IsEnabled(CustomComboPreset.WhiteMageDotRefresh))
 		{
```

Comment:
This line and the one below - for consistency with the project's style, please place open braces on the same line as the statement they apply to.